### PR TITLE
Adapting e2e upgrade test to be handled by KEB

### PR DESCRIPTION
**Pull request by jasiu001**
**Description**

Changes proposed in this pull request:

- adapting e2e upgrade test to be handled by KEB

**Related issue(s)**
#137

### CODE REVIEW

File: tests/e2e/provisioning/pkg/client/broker/update_broker_client.go [hunk 1]
```diff
@@ -0,0 +1,195 @@
+package broker
```
Comment (polskikiel) [inline tests/e2e/provisioning/pkg/client/broker/update_broker_client.go:1]: file could be renamed to `upgrade_broker_client`

File: tests/e2e/provisioning/test/e2e_upgrade_test.go [hunk 2]
```diff
@@ -39,12 +38,17 @@ func Test_E2E_Upgrade(t *testing.T) {
...
 	require.NoError(t, err)
 
-	ts.log.Info("Starting upgrade")
-	upgradeOperationID, err := ts.upgradeSuite.upgradeClient.UpgradeRuntimeToVersion(ts.upgradeSuite.UpgradeKymaVersion)
+	ts.log.Infof("Fetch runtimeID from CLD endpoint based on instanceID: %s", ts.InstanceID)
+	runtimeID, err := ts.upgradeSuite.upgradeClient.FetchRuntimeID(ts.InstanceID)
+	require.NoError(t, err)
+
+	ts.log.Infof("Starting upgrade runtime with ID: %s", runtimeID)
+	orchestrationID, err := ts.upgradeSuite.upgradeClient.UpgradeRuntime("e31590e0-3c32-4445-a4a7-4a90eb28e859")
```
Comment (polskikiel) [inline tests/e2e/provisioning/test/e2e_upgrade_test.go:46]: runtimeID here ->

File: tests/e2e/provisioning/pkg/client/broker/update_broker_client.go [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+	if err != nil {
+		return "", errors.Wrap(err, "while decoding upgrade response")
+	}
+
+	return upgradeResponse.OrchestrationID, nil
+}
+
+func (c *UpgradeClient) FetchRuntimeID(instanceID string) (string, error) {
+	var runtimeID string
+	err := wait.Poll(3*time.Second, 60*time.Second, func() (bool, error) {
```
Comment (polskikiel) [inline tests/e2e/provisioning/pkg/client/broker/update_broker_client.go:78]: 60s->1m

File: tests/e2e/provisioning/test/e2e_upgrade_test.go (hunk 2 above)
Comment (jasiu001) [reply tests/e2e/provisioning/test/e2e_upgrade_test.go:46]: ups... test residue

**Review by polskikiel — APPROVED** [review]

**Pull request merged**